packages/radar/index.d.ts

```diff
@@ -12,16 +12,28 @@ import { OrdinalColorsInstruction, InheritedColorProp } from '@nivo/colors'
 import { LegendProps } from '@nivo/legends'
 
 declare module '@nivo/radar' {
-    type IndexByCustomFunctiono<D = any> = (datum: D) => string | number
-    type GridLabelCustomFunction = (...args: any[]) => string | JSX.Element
-    type CustomDotSymbol = (...args: any[]) => React.ReactNode
-    type CustomDotLabel = (...args: any[]) => React.ReactNode
-    type CustomFormatter = (...args: any[]) => React.ReactNode
+    export type GridLabelProps = {
+        id: string
+        anchor: "start" | "middle" | "end"
+        angle: number
+    }
+    export type DotSymbolProps = {
+        size: number
+        color: InheritedColorProp
+        borderWidth: number
+        borderColor: InheritedColorProp
+    }
+
+    type IndexByCustomFunction<D = any> = (datum: D) => string | number
+    export type CustomGridLabel = React.FC<GridLabelProps>
+    export type CustomDotSymbol = React.FC<DotSymbolProps>
+    export type CustomDotLabel = (...args: any[]) => React.ReactNode
```

Comment:
I think these should be strictly typed too, I never used them and I honestly don't have enough knowledge about them to type them.

Reply:
I agree, but I guess we can add this later, what you updated is already a good addition.